float64 gain        # T = gain*C + offset
float64 offset      # T: time in seconds, C: count on triggerbox hardware
